src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,164 @@
+
+I left twitter recently, shortly after a hasty user poll resulted in the reinstatement of former (and twice-impeached) president Trump's account.
+
+<Tweet align="center" tweetLink="irreverentmike/status/1594455660040572928" />
+
+Since then, another hasty poll has resulted in an absolutely staggering policy change: all\* accounts that were previously suspended for violating Twitter's rules will be reinstated.
+
+<Tweet align="center" tweetLink="elonmusk/status/1595473875847942146" />
+
+Pandora's box has been opened, and the monsters are coming back. Twitter is now a platform for hate speech, misinformation, and disinformation. It's will become a cesspool of conspiracy theories, and a breeding ground for countless nightmares and real-world dangers.
+
+This is a very real, very observable truth: countless racists, abusers, and other dangerous people are being given "amnesty" to come back the platform.
+
+I've seen many others abandon their twitter accounts in the wake of these changes, but -- friends -- I'm honestly shocked that so many people choose to remain.
+
+Why are so many people still on twitter, administer electric shocks to the collective public by keeping their accounts alive?
+
+I'm reminded by the words of Stanley Milgram, in his reflection on the results of his now-infamous experiment:
+
+> Ordinary people, simply doing their jobs, and without any particular hostility on their part, can become agents in a terrible destructive process. Moreover, even when the destructive effects of their work become patently clear, and they are asked to carry out actions incompatible with fundamental standards of morality, relatively **few people have the resources needed to resist authority.**
+
+For many people, Twitter was a place to connect with friends, family, and colleagues. I know many people with follower counts in the tens of thousands, who have built a career around their twitter presence. I can understand a deep hesitancy to abandon a platform that has been so central to your life.
+
+But fam, seriously -- **if not now, when?**
+
+## A call to action
+
+Friends, it's time to leave Twitter. Shutter your accounts, delete the app, and move on. I know it's hard, but it's time. Every moment you spent on Twitter is feeding legitimacy to a platform that is enabling abusers, endangering global politics, and destroying the fabric of our society.
+
+It is no longer acceptable to stay aboard the ship as it sinks - to stay active on Twitter despite their new policies and leadership is a vanity, a bath in your own privilege.
+
+Bring your voice and your conversations elsewhere, and send the message that with its current leadership, Twitter is no longer a place for you.
```

Comment:
I'm also concerned about the precedent this sets if Musk succeeds. Will every tech company be bought, gutted, employees thrown around like trash?